website/pages/changelog.tsx

```diff
@@ -1,8 +1,35 @@
 import React from "react"
-import MarkdownContent from "../../CHANGELOG.md"
+import { MDXRemote } from "next-mdx-remote"
+import { MDXComponents } from "components/mdx-components"
+import { InferGetStaticPropsType } from "next"
+import Layout from "layouts"
+import { loadMdx } from "utils/load-mdx"
 
-function Changelog() {
-  return <MarkdownContent />
+function Changelog({
+  mdxSource,
+  frontMatter,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+  return (
+    <Layout frontMatter={frontMatter}>
+      <MDXRemote {...mdxSource} components={MDXComponents} />
+    </Layout>
+  )
 }
 
+export async function getStaticProps() {
+  const changelogPath = ["..", "CHANGELOG.md"].join("/")
+  const page = await loadMdx(changelogPath)
+  if (!page) {
+    console.warn(`No changelog found`)
```

Comment:
What do you think about breaking the build if no Changelog is found? 
The vercel build log is easily overlooked in a PR, and I think this would help us to notice the error.

Reply:
Good idea! added a check if the`md` file is available in `loadMdx` and break otherwise. https://github.com/chakra-ui/chakra-ui/commit/e3b3d49aaf5c11e33ddcf177c1554645b2d5aa53